infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesEnvironmentProvisioner.java

```diff
@@ -123,6 +131,13 @@ public KubernetesEnvironmentProvisionerImpl(
     @Traced
     public void provision(KubernetesEnvironment k8sEnv, RuntimeIdentity identity)
         throws InfrastructureException {
+      provision(k8sEnv, identity, null);
```

Comment:
namespace is null? How it will work?

Reply:
it will work exaclty as before :) since namespce neede is only for one provisioner, and it just do nothing if there is null

Reply:
when is it null ? Do we explicitly want it in some cases?